src/Ninja/G1CP/Content/Fixes/Session/fix168_GorNaBarBecomeTemplar.d

```diff
@@ -0,0 +1,30 @@
+/*
+ * #168 Gor Na Bar offers to become a templar to anyone
+ */
+func int G1CP_168_GorNaBarBecomeTemplar() {
+    if (G1CP_IsFunc("Tpl_1400_GorNaBar_VICTORY_Condition", "int|none"))
+    && (G1CP_IsIntConst("GIL_NOV", 0)) {
+        HookDaedalusFuncS("Tpl_1400_GorNaBar_VICTORY_Condition", "G1CP_168_GorNaBarBecomeTemplar_Hook");
+        return TRUE;
+    } else {
+        return FALSE;
+    };
+};
+
+/*
+ * This function intercepts the dialog condition to introduce more conditions
+ */
+func int G1CP_168_GorNaBarBecomeTemplar_Hook() {
+    G1CP_ReportFuncToSpy();
+
+    // Define possibly missing symbols locally
+    const int GIL_NOV = 12;
```

Comment:
I wonder if the constant should rather be retrieved here. The value (12) may have been changed in the scripts. How have we done this in similar fixes?

Reply:
From what I could find we always defined the guild constants.

Reply:
Ah. I wonder what the idea was there and if I am missing something now. Do you happen do have a (few) examples at hand at the moment? I couldn't find one so fast.

Reply:
Ignoring the test suite, I found that we retrieved the guild in the fixes of #79 and also #32. I could not find any other fixes regarding guilds.

Reply:
We can leave the above line untouched but add a [block like found here](https://github.com/AmProsius/gothic-1-community-patch/blob/1d4b915c0f50c75e0b20eb0971907cc62b1929c7/src/Ninja/G1CP/Content/Fixes/Session/fix032_GornAttackFreeMine.d#L87-L96).

```D
    // Update the symbols given they exist (once)
    const int updated = FALSE;
    if (!updated) {
        GIL_NOV = G1CP_GetIntConst("GIL_NOV", 0, GIL_NOV);
        updated = TRUE;
    };
```

I propose to use this once-block here, because the condition function is called multiple times. Retrieving the constant only once may be beneficial to performance (although the impact will not be noticeable).